examples/simple-job-script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#SBATCH -J dummy_job
+#SBATCH -t 60
+
+echo "Executing simple job script"
+
+echo "Simple output from $HOSTNAME" > /nfs/simple-output.txt
```

Comment:
I'm not a big fan of using the absolute path in this case because it ignores the submission directory, which can lead to file conflicts. 
```suggestion

 echo "Simple output from $HOSTNAME" > ./simple-output.txt
```

Reply:
Updated in commit e2e8554.